extension/js/utils.js

```diff
@@ -0,0 +1,13 @@
+/* exported areInputIconsEnabled */
+
+// eslint-disable-next-line no-redeclare
```

Comment:
Do we still  need the `eslint-disable`?

Reply:
Yeah without it eslint says `'areInputIconsEnabled' is already defined as a built-in global variable. (no-redeclare)`